serverless.yml

```diff
@@ -10,7 +10,7 @@ custom:
   esbuild:
     bundle: true
     minify: true
-    target: node16
+    target: node18
```

Comment:
According to the document from serverless esbuild plugin, we do not need to set this as it will be dynamically set based on provider runtime: [https://github.com/floydspace/serverless-esbuild#supported-runtimes](https://github.com/floydspace/serverless-esbuild#supported-runtimes)

Reply:
In addition, we should add `exclude: ['@aws-sdk']` option to remove AWS-SDK v3 from our lambda bundle.

Reply:
Interesting, thanks